tests/fixtures/entity-viewer/transcript.ts

```diff
@@ -69,10 +84,37 @@ export const createTranscript = (
       length: transcriptSlice.location.length
     },
     spliced_exons: createSplicedExons(transcriptSlice, exons),
+    metadata: createTranscriptMetadata()
+  };
+};
+
+export const createNonCodingTranscript = (
+  fragment: Partial<NonCodingTranscript> = {}
+): NonCodingTranscript => {
+  const transcript = createCommonTranscriptFields();
+  const transcriptSlice = transcript.slice;
+  const productGeneratingContext =
+    createNonCodingProductGeneratingContext(transcriptSlice);
+
+  return {
+    ...transcript,
+    product_generating_contexts: [productGeneratingContext],
+    ...fragment
+  };
+};
+
+export const createProteinCodingTranscript = (
+  fragment: Partial<ProteinCodingTranscript> = {}
+): ProteinCodingTranscript => {
+  const transcript = createCommonTranscriptFields();
+  const transcriptSlice = transcript.slice;
```

Comment:
Why define this in a new variable when we can use it as it is below?

Reply:
Done. The reason it was declared as a variable was because i thought we will use it again but thats not the case in that function.